src/components/action-buttons.component.tsx

```diff
@@ -21,25 +21,23 @@ import PauseDispenseForm from "../forms/pause-dispense-form.component";
 import CloseDispenseForm from "../forms/close-dispense-form.component";
 
 interface ActionButtonsProps {
-  medicationRequest: MedicationRequest;
-  associatedMedicationDispenses: Array<MedicationDispense>;
+  medicationRequestBundle: MedicationRequestBundle;
```

Comment:
I've tried to simplify this by creating a new convenience object that combines a medication request with it's associated dispenses